README.md

```diff
@@ -1,34 +1,109 @@
+```git fetch upstream```
 
-Once complete, commit and push the changes to your forked branch. Create a pull request from the forked branch to the main repo to merge your changes into the main repo.
-[Please see this document for more information](https://github.com/CoolProp/CoolProp/wiki/Contributing:-git-development-workflow).
+```git checkout master```
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+```git merge upstream/master```
+
+You are now ready to develop your connector in your own branch.
+
+### Submitting to the Open Source Repository
+
+Contributions to the open source repository are made through pull requests. 
+Prior to submitting a pull request, ensure that your pull request does not contain any sensitive or specific information, for example Client IDs or Client Secrets. 
+Any sensitive values can be replaced with fake or dummy values for the purposes of submission as long as it is clearly indicated. 
+Also, ensure that the readme.md of the connector is updated with the latest information, or created for new connector submissions. 
+An example of a clear, structured, readme.md can be found in the [Azure Key Vault](https://github.com/microsoft/PowerPlatformConnectors/tree/master/custom-connectors/AzureKeyVault) connector repository. 
+Include this completed `readme.md` in same connector directory which contains the artifacts. 
+
+#### Custom Connectors
+
+Updates to an existing custom connector can be made through a simple pull request to update the custom connector files.
+
+For new custom connectors, create a directory under the ```custom-connectors``` directory and place the connector files in the sub-folder. Ensure that a clear, structured, readme.md is included. 
+
+#### Certified Connectors
+
+Updates to certified connectors must first be made through a pull request to the ```dev``` branch. 
+Once in the ```dev``` branch, the certified connector owner can submit the connector for certification and deployment throughout the Power Platform. 
+Once certified, the Microsoft Certification team will handle merging the updates from ```dev``` to ```master```. 
+
+Updates to an existing certified connector can be made through a simple pull request to the ```dev``` branch to update the certified connector files. 
+This will be reviewed by the connector owner. Once a PR has been merged to the ```dev``` branch, the connector owner can submit the connector for certification through the Connector certification tab in [ISV Studio](https://isvstudio.powerapps.com). 
```

Comment:
These two sections appear to have a somewhat similar message but are phrased differently. Are we addressing two separate audiences here, the connector owner and a third party member of the community looking to contribute? It's not immediately apparent to me why there are two distinct sections here.

Reply:
That was my glossy mind writing this out too late at night - good catch and resolved.